CDP4WebServices.API/Modules/10-25/ApiBase.cs

```diff
@@ -867,19 +900,31 @@ protected string GenerateRandomToken()
                 // use the file hash value to easily identify the multipart content for each respective filerevision hash entry
                 binaryContent.Headers.Add(
                     ContentDispositionHeader,
-                    string.Format("attachment; filename={0}", folderPath + ".zip"));
+                    $"attachment; filename={folderPath + ".zip"}");
                 binaryContent.Headers.Add(ContentLengthHeader, fileSize.ToString());
                 content.Add(binaryContent);
 
                 // stream the multipart content to the request contents target stream
                 content.CopyToAsync(targetStream);
+
+                this.AddMultiPartMimeEndpoint(targetStream);
             }
             finally
             {
                 this.FileArchiveService.DeleteFileStructureWithArchive(folderPath);
             }
         }
 
+        /// <summary>
+        /// add ending line according to https://www.w3.org/Protocols/rfc1341/7_2_Multipart.html
+        /// </summary>
+        /// <param name="targetStream">The <see cref="Stream"/>where to add the end poit to</param>
+        private void AddMultiPartMimeEndpoint(Stream targetStream)
```

Comment:
can we make the parameter a `ref` parameter to make it obvious we are chaining the parameter `targetStream` in this mehtod?